pkg/kubectl/stop.go

```diff
@@ -78,19 +78,26 @@ func (reaper *ReplicationControllerReaper) Stop(namespace, name string) (string,
 	return fmt.Sprintf("%s stopped", name), nil
 }
 
-func (reaper *PodReaper) Stop(namespace, name string) (string, error) {
+func (reaper *PodReaper) Stop(namespace, name string, gracePeriodSec int) (string, error) {
 	pods := reaper.Pods(namespace)
 	_, err := pods.Get(name)
 	if err != nil {
 		return "", err
 	}
-	if err := pods.Delete(name); err != nil {
-		return "", err
+	if gracePeriodSec >= 0 {
```

Comment:
Leave Delete as is, but take name and api.DeleteOptions (similar to the rest method).

----- Original Message -----
> >  	pods := reaper.Pods(namespace)
> >  	_, err := pods.Get(name)
> >  	if err != nil {
> >  		return "", err
> >  	}
> > -	if err := pods.Delete(name); err != nil {
> > -		return "", err
> > +	if gracePeriodSec >= 0 {
> 
> You mean renaming Delete to DeleteWithOptions?
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/GoogleCloudPlatform/kubernetes/pull/7432/files#r29274548